tools/dashboard-plugin-template/webpack.config.js

```diff
@@ -91,8 +92,8 @@ module.exports = (_env, argv) => {
                     type: "asset/resource",
                 },
                 {
-                    // never tree shake ext so that the version here is also used in the plugins
-                    test: (modulePath) => /sdk-ui-ext/.test(modulePath),
+                    // never tree shake ext or dashboard so that the version here is also used in the plugins
+                    test: (modulePath) => /sdk-ui-(ext|dashboard)/.test(modulePath),
```

Comment:
why not ext? we should defer to the one that is used in KD. or is this the thing you want to look into?